pages/friendlist.js

```diff
@@ -4,9 +4,14 @@ import Logo from "@/Icons/Logo";
 import { theme } from "@/styles";
 import Navigation from "@/components/Navigation";
 import CardForm from "@/components/CardForm";
+import useAuthentication from "./api/auth/useAuthentication";
 import UserProfile from "@/components/UserProfile";
 
 export default function FriendList({ onSubmit, setIsEditMode }) {
+  const { authenticated, loading, session } = useAuthentication();
+  const userId = session?.user?.id;
+  const id = "65fd9d760c057e6bcdb880cd";
```

Comment:
Whilst i understand the need for test ids while building the project, i am not sure we want this merged into main. No need to make any changes this time.